### PR TITLE
Add myself, JCount to maintainers lists

### PR DESCRIPTION
**Pull request by woodruffw**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

This is a pretty big change, might take a while to churn through the CI :smile:

### CODE REVIEW

**Comment by woodruffw** [comment]
Added @JCount as well, on request :tada:

**Comment by MikeMcQuaid** [comment]
Apologies you two weren't added until now! Also added @fxcoudert and sorted the list so it's easier to check who is missing in future.

File: docs/brew.1.html [hunk 1]
```diff
@@ -791,7 +791,7 @@ <h2 id="AUTHORS">AUTHORS</h2>
 
 <p>Homebrew's lead maintainer is Mike McQuaid.</p>
 
-<p>Homebrew's current maintainers are Alyssa Ross, Andrew Janke, Baptiste Fontaine, Alex Dunn, ilovezfs, Josh Hagins, JCount, Misty De Meo, Tomasz Pajor, Markus Reiter, Tim Smith, Tom Schoonjans, Uladzislau Shablinski and William Woodruff.</p>
+<p>Homebrew's current maintainers are Alyssa Ross, Andrew Janke, Baptiste Fontaine, [Alex Dunn](https://github.</p>
```
Comment (zmwangx) [inline docs/brew.1.html]: @MikeMcQuaid List chopped off here. Same for `brew.1`.

Comment (JCount) [inline docs/brew.1.html]: Addressed already in #2186.

**Pull request merged**